src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class AddCommand extends Command {
+    public AddCommand(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Add a Deadline/ Event/ To-do task to the task list, save the list to disk and display to user.
+     *
+     * @param  tasks   the task list
```

Comment:
A little awkward extra spacings around here should be:
@param tasks the task list

same with the other parameters